R/weightedMeanByID.R

```diff
@@ -0,0 +1,86 @@
+#' @title Aggregate different rows assigned to the same ID by calculating a 
```

Comment:
General comment: We try to keep lines below 80 characters whenever possible (may not always be possible, but in many cases it is, and it makes the code easier to read)